src/net_stat.cc

```diff
@@ -361,7 +355,7 @@ void print_wireless_mode(struct text_object *obj, char *p, int p_max_size)
 	struct net_stat *ns = (struct net_stat *)obj->data.opaque;
 
 	if (!ns)
-		return;
+	return;
```

Comment:
Looks like a bunch of formatting changes. Can you either leave it as is, or convert these to:

```c
if (!ns) {
    return;
}
```

Reply:
Yes as I stated I used Eclipse auto-indent on the whole file by habit. I'll just take the original files, cautiously re-add my modifications and do a new commit.